test/fixtures/rippled/account-info.json

```diff
@@ -17,7 +17,36 @@
       "TransferRate": 1002000000,
       "WalletLocator": "00000000000000000000000000000000000000000000000000000000DEADBEEF",
       "index": "396400950EA27EB5710C0D5BE1D2B4689139F168AC5D07C13B8140EC3F82AE71",
-      "urlgravatar": "http://www.gravatar.com/avatar/23463b99b62a72f26ed677cc556c44e8"
+      "urlgravatar": "http://www.gravatar.com/avatar/23463b99b62a72f26ed677cc556c44e8",
+      "signer_lists": [
```

Comment:
How was this test fixture generated?

Reply:
I copied a `SignerList` ledger entry from the altnet
`curl -X POST -d '{ "method": "ledger_entry", "params": [ { "ledger_index": "validated", "index": "5A9373E02D1DEF7EC9204DEB4819BA42D6AA6BCD878DC8C853062E9DD9708D11", "binary":false}]}' http://client.altnet.rippletest.net:51234`